include/onnxruntime/core/session/onnxruntime_session_options_config_keys.h

```diff
@@ -119,3 +119,22 @@ static const char* const kOrtSessionOptionsConfigDynamicBlockBase = "session.dyn
 // "0": in some cases warnings will be logged but processing will continue. The default.
 // May be useful to expose bugs in models.
 static const char* const kOrtSessionOptionsConfigStrictShapeTypeInference = "session.strict_shape_type_inference";
+
+// SessionOption 'fixed_point_requant_on_arm' controls the requantization method on ARM devices.
+// Requantization is computed with formula:
+//     v = round(clamp(S * (I - Z), min, max))
+// where v is the target value with type TOutput, which is either int8_t or uint8_t
+//       I is the input value with type int32_t
+//       S is the scale with type float
+//       Z is the zero point with type same as TOutput.
+//       min is the minimum value of type TOutput.
+//       max is the maximum value of type TOutput.
```

Comment:
Great comments, very informative! 
1. This is a performance improvement, yet the option is by default "off", due to rounding errors? Would you consider specifying the reason for this?
2. The same trick might work in other CPUs too. Since it's default off, maybe removing ARM from name?

Reply:
yes, it is turned off by default for rounding. It rounds half to up, however onnx specs requires to round half to even. Will add the info.

There is no plan to support it on x86. keeping arm here can avoid the confusion that users think x86 supports similar option.